src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -520,14 +520,43 @@ void xocl_mm_update_usage_stat(struct xocl_drm *drm_p, u32 ddr,
 	drm_p->mm_usage_stat[ddr]->bo_count += count;
 }
 
+int xocl_mm_insert_node_range(struct xocl_drm *drm_p, u32 mem_id,
+			      struct drm_mm_node *node, u64 size)
+{
+	uint64_t start_addr = 0;
+	uint64_t end_addr = 0;
+	struct mem_topology    *topology = NULL;
```

Comment:
Not clear why so much spaces after `mem_topology`

Reply:
Will fix it.